pkg/config/config.go

```diff
@@ -71,6 +71,7 @@ type Config struct {
 	Keys           map[string]string        `yaml:"keys,omitempty"`
 	Region         string                   `yaml:"region,omitempty"`
 	SignalRelay    SignalRelayConfig        `yaml:"signal_relay,omitempty"`
+	PSRPC          PSRPCConfig              `yaml:"room_service,omitempty"`
```

Comment:
the yaml config doesn't match config key. `room_service` seems confusing as a config option